core/src/it/scala/org/ensime/indexer/SearchServiceSpec.scala

```diff
@@ -158,6 +158,22 @@ class SearchServiceSpec extends EnsimeSpec
     hits.head shouldBe "org.example2.Baz"
   }
 
+  it should "return user created classes first" in withSearchService { implicit service =>
+    val hits = service.searchClasses("File", 10).map(_.fqn)
+    hits.head should startWith("org.example.File")
+    hits.length shouldBe 8
```

Comment:
asserting on length is going to be very fragile across systems, just assert the first hit and that the `java.io.File` result is in the response *somewhere* (likewise below)